### ISSUE TITLE
Accessibility: on systems with larger DPI scaler (e.g. tablet-like form factor or user with poor sight) start screen might be cut off

### ISSUE THREAD

**Issue by ell1e** [deltachat/deltachat-desktop#4063]
It seems like on systems with a larger DPI scaler given a relatively normal or smaller screen size, e.g. tablet-like form factor where people may often still run Desktop Linux, or on computers of user with poor sight, the delta chat start screen seems to be cut off. It would be nice if it was more responsive and scrollable to work around that.

- Operating System (Linux/Mac/Windows/iOS/Android): postmarketOS Edge x64 desktop (Alpine Linux based), using a regular KDE Desktop and not any kind of mobile desktop
- Delta Chat Version: v1.140.2 (latest on flathub)
- Expected behavior: start screen and about screen aren't cut off
- Actual behavior: it's cut off:
  ![Screenshot_20240803_144258](https://github.com/user-attachments/assets/b0d71599-3d67-43d2-ad0c-5fb4fd0a6220)
  The about screen can't fully scroll to the buttom either:
  ![Screenshot_20240803_144710](https://github.com/user-attachments/assets/1c5bef8a-6eaf-42a6-9582-a24b5442acd5)

- Steps to reproduce the problem:

  1. Set DPI scaler somewhat high on a normal sized or slightly smaller sized laptop or tablet screen. E.g. an ultraportable Ultrabook or something like the GPD Win or like the Steam deck or a normal laptop with poor eyesight user.
  2. Start delta chat
  3. Observe that it doesn't seem to fit the screen fully
- Screenshots: see above
- Logs:

**Comment by Simon-Laux** [comment]
What's your window/screen resolution in pixels?

**Comment by ell1e** [comment]
800x1280 with DPI scaler 1.75

**Comment by WofWca** [comment]
I believe HTML/CSS-wise we're good. It appears like the window itself goes beyond the screen bounds in the original post.

https://github.com/user-attachments/assets/6a9521f9-0da8-485b-ab2a-06f884352f65

On Windows I cannot resize the window to shorter than 400px. I think we set the `minHeight` here:

https://github.com/deltachat/deltachat-desktop/blob/a5c02433976909fe01a64f3049c96d0d101ae9d8/packages/target-electron/src/application-constants.ts#L43-L44

Shall we just raise it? Do we have a reason to have it no lower than 450?

**Comment by Simon-Laux** [comment]
I wonder why these values are called with system dpi scaler. maybe we can lower even further or even remove the min- screen size. (but then people shouldn't complain if the interface is not working on 100x100px for example...)

> Do we have a reason to have it no lower than 450?

That was the minimum-somewhat-usable size before the introduction of small screen mode.

**Comment by maxphilippov** [comment]
> Shall we just lower it? Do we have a reason to have it no lower than 450?

DEFAULT_WIDTH in Dialog.tsx is set to 500. I did drag it over into a #4006. I'm not sure that might cause problems and I can't check right now, just saying.
